demisto_sdk/commands/pre_commit/hooks/mypy.py

```diff
@@ -0,0 +1,6 @@
+from demisto_sdk.commands.pre_commit.hooks.hook import Hook
+
+
+class MypyHook(Hook):
+    def prepare_hook(self, python_version: str, **kwargs):
```

Comment:
looks like either an `__init__`, or a static method. Which is it?

Reply:
The init is on the super class.
This class is responsible for preprocess and postprocess the hook.
Ideally, we will add a `postprocess` method to post process the results